### PR TITLE
fixes bug 1383067 - redo symbols-urls conf in BreakpadStackwalker2015

### PR DESCRIPTION
**Pull request by willkg**
This drops the private-symbols-url and public-symbols-url configuration
variables for a single symbols-urls variable that takes a comma-delimited
list of strings.

Now we can add arbitrary number of urls in the order we want them
checked.

### CODE REVIEW

**Comment by willkg** [comment]
Before this lands, we need to do configuration changes in -stage. Those are discussed in https://bugzilla.mozilla.org/show_bug.cgi?id=1383067#c7 .

To test this, do the following:

1. Add this to `my.env` to make the `BreakpadStackwalkerRule2015` rule chatty:
   
   ```
   processor.raw_to_processed_transform.BreakpadStackwalkerRule2015.chatty=1
   ```
2. Run the processor. You'll see this line in the copious configuration logging:
   
   ```
   2017-12-01 15:51:40,824 INFO - processor -  - MainThread - processor.raw_to_processed_transform.BreakpadStackwalkerRule2015.symbols_urls: ['https://s3-us-west-2.amazonaws.com/org.mozilla.crash-stats.symbols-public/v1', 'https://s3-us-west-2.amazonaws.com/org.mozilla.crash-stats.symbols-private/v1']
   ```
3. Process a crash. Because the rule is chatty, it'll tell you how it ran mdsw. Should be something like this:
   ```
   processor_1      | 2017-12-01 15:37:18,049 DEBUG - processor -  - Thread-2 - External Command: timeout -s KILL 30 /stackwalk/stackwalker --raw-json /tmp/b4b52c79-3c02-4341-bbc9-9205f0171130.Thread-2.TEMPORARY.json --symbols-url https://s3-us-west-2.amazonaws.com/org.mozilla.crash-stats.symbols-public/v1 --symbols-url https://s3-us-west-2.amazonaws.com/org.mozilla.crash-stats.symbols-private/v1 --symbols-cache /tmp/symbols/cache --symbols-tmp /tmp/symbols/tmp /tmp/b4b52c79-3c02-4341-bbc9-9205f0171130.upload_file_minidump_browser.TEMPORARY.dump 2> /dev/null
   ```
   Both the org.mozilla.crash-stats.symbols-public and org.mozilla.crash-stats.symbols-private urls should be in there in that specific order.

File: socorro/processor/breakpad_transform_rules.py [hunk 1]
```diff
@@ -317,25 +305,27 @@ def _execute_external_process(self, command_line, processor_meta):
 
     def expand_commandline(self, dump_file_pathname, raw_crash_pathname):
         """Expands the command line parameters and returns the final command line"""
-
         # NOTE(willkg): If we ever add new configuration variables, we'll need
         # to add them here, too, otherwise they won't get expanded in the
         # command line.
 
+        symbols_urls = ' '.join([
+            '--symbols-url %s' % url.strip()
```
Comment (peterbe) [inline socorro/processor/breakpad_transform_rules.py]: Drive-by-comment; Shouldn't that be `'--symbols-url "%s"'` just in case URL has a character in it that affects the parameter passing.

Comment (willkg) [reply socorro/processor/breakpad_transform_rules.py]: Yes, it should. Thank you!

**Comment by willkg** [comment]
Thank you!
